nixos/modules/tasks/network-interfaces.nix

```diff
@@ -565,6 +582,7 @@ in
             example = 100;
             type = types.nullOr types.int;
             description = ''
+              DEPRECATED, use `driverOptions`.
```

Comment:
This is not true, at least not using the scripted networking. This breaks bonding for me. Where do you merge the deprecated settings?